content/en/docs/admin/admission-controllers.md

```diff
@@ -316,23 +314,24 @@ In any case, the annotations are provided by the user and are not validated by K
 
 ### Initializers (alpha)
 
+This admission controller is introduced in v1.7.
```

Comment:
Again, is the version relevant? 1.7 will be the oldest supported version when 1.11 ships, if I understand it correctly.